src/app/views/resources/Resources.scss

```diff
@@ -58,12 +60,40 @@
       padding: 0 20px;
     }
   }
+
+  .resource-border {
+    border-right: 1px solid white;
+    border-width: medium;
+  }
+
+  .resource-component {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    text-align: center;
```

Comment:
The flex design obscures the fact that the h4 headings are still text-align left which becomes visible on narrower views.